src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -746,6 +745,21 @@ function barSeriesMapFn(seriesOrig: any) {
     return series;
 }
 
+function getHeatMapHoverColor(config: any) {
+    const dataClasses = get(config, ['colorAxis', 'dataClasses'], null);
+    let resultColor = 'rgb(210,210,210)';
```

Comment:
what about a constant put somewhere near to the HEATMAP_BLUE_COLOR_PALETTE constant?